.github/workflows/workflow.yml

```diff
@@ -83,7 +81,7 @@ jobs:
       - uses: actions/upload-artifact@v2
         if: always()
         with:
-          name: cypress-results-node-${{ matrix.node-version }}-os-${{ matrix.os }}
+          name: cypress-results-node-os-${{ matrix.os }}
```

Comment:
👍

Reply:
Note: the reason this was removed is because Cypress does not allow `*` in there.